java/ql/lib/semmle/code/java/security/Files.qll

```diff
@@ -0,0 +1,72 @@
+/** Provides classes and predicates to work with File objects. */
+
+import java
+import semmle.code.java.dataflow.ExternalFlow
+
+private class CreateFileSinkModels extends SinkModelCsv {
+  override predicate row(string row) {
+    row =
+      [
```

Comment:
Should this also cover the `java.io.File` methods (unless they are covered somewhere else)?
- [`createTempFile`](https://docs.oracle.com/en/java/javase/16/docs/api/java.base/java/io/File.html#createTempFile(java.lang.String,java.lang.String,java.io.File)) (`directory` argument)
- `createNewFile()`, `mkdir()`, `mkdirs()`: Receiver is the sink
- `renameTo(File)`: Argument is the sink